docs/how-to-create-model.md

```diff
@@ -125,6 +125,35 @@ int main(int argc, char *argv[]) {
 ```
 
 ## Build and Run "MyApp" application
+
+
+### Add  meson.build file under `MyApp/jni`
+
+In order to build MyApp application, it is required to add `meson.build` for it. To this end, create the build file under `Applications/MyApp/jni/`.
+Here is an example of `meson.build` file of `MyApp` application.
+
+```
+myapp_sources= [
+    'main.cpp',
+    cifar_path / 'cifar_dataloader.cpp'
+]
+
+myapp_dependencies = [app_utils_dep,
+  nntrainer_dep,
+  nntrainer_ccapi_dep
+]
+
+e = executable('nntrainer_myapp',
+  myapp_sources,
+  include_directories: [include_directories('.'), cifar_include_dir],
+  dependencies: myapp_dependencies,
+  install: get_option('install-app'),
+  install_dir: application_install_dir
+)
+```
+
```

Comment:
How about moving this explain to below contents

```
Add the following code at the end of the "Applications/meson.build": "subdir('MyApp/jni')".
...
subdir('MyApp/jni')

```

Line 25

Reply:
Absolutely! it would be more readable. Thnx.